### PR TITLE
droidcam: Add version 6.3.3

### PR DESCRIPTION
**Pull request by lmartinez-mirror**
DroidCam is a desktop application that uses Android Debug Bridge to connect your Android device's camera and microphones and use them as a webcam.

It is freeware, however there is also a paid version available with more quality of life features.

### CODE REVIEW

File: bucket/droidcam.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "64bit": {
+            "url": "https://files.dev47apps.net/win/DroidCam.Setup.6.3.3.exe#/dl.7z",
+            "hash": "sha512:48f4b5bbe73569df513137a254a16b90a0008f764c9d79aae2ca748fbc54cbf4544e9faa4b0cb5fcef81f693ccde175c696a5fad5fa2176cbc7926297f5c19e1"
+        }
+    },
+    "notes": [
+        "For instructions on connecting, visit https://www.dev47apps.com/droidcam/connect/.",
+        "You may need to install the appropriate USB driver for your Android device to get this to work.",
+        "Please visit https://developer.android.com/studio/run/oem-usb#Drivers for more information."
+    ]
```
Comment (Ash258) [inline bucket/droidcam.json:23]:
```suggestion

    "version": "6.3.3",
    "description": "Allows to use phone as a webcam for PC.",
    "homepage": "https://dev47apps.com/",
    "license": "Freeware",
    "notes": "See 'https://www.dev47apps.com/droidcam/help/' for configuration steps.",
    "suggest": {
        "vcredsit": "vcredist"
    },
    "architecture": {
        "64bit": {
            "url": "https://files.dev47apps.net/win/DroidCam.Setup.6.3.3.exe#/dl.7z",
            "hash": "0fb1f9b6526cbf0175d152017a99c050a87746948360c807297075117dd233e8"
        }
    },
    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\vc_red*\", \"$dir\\Uninstall*\" -Recurse",
    "shortcuts": [
        [
            "DroidCamApp.exe",
            "DroidCam"
        ]
    ],
    "checkver": {
        "url": "https://www.dev47apps.com/droidcam/windows/",
        "regex": "v([\\d.]+)\\s+Download"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://files.dev47apps.net/win/DroidCam.Setup.$version.exe#/dl.7z"
            }
        }
    }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Why do you depend on adb?

A

File: bucket/droidcam.json (hunk 1 above)
Comment (lmartinez-mirror) [reply bucket/droidcam.json:23]: DroidCam has a soft dependency on adb for Android devices, its target audience. Without it it will not work.

Comment (Ash258) [reply bucket/droidcam.json:23]: It has it already...

Comment (lmartinez-mirror) [reply bucket/droidcam.json:23]: Apologies, I didn't see that detail.

**Review by Ash258 — COMMENTED** [review]
Manifest works good, but I cannot verify the functionality

**Comment by earnestma** [comment]
> Manifest works good, but I cannot verify the functionality

Tested, works as usual with Droidcam v6.8.2 from play store

**Pull request closed without merging**